README.md

```diff
@@ -1,4 +1,8 @@
-[![Linting](https://img.shields.io/badge/PEP8%20by-Hound%20CI-a873d1.svg)](https://houndci.com) [![Coverage Status](https://coveralls.io/repos/github/shotgunsoftware/tk-core/badge.svg?branch=master)](https://coveralls.io/github/shotgunsoftware/tk-core?branch=master) [![Build Status Linux](https://secure.travis-ci.org/shotgunsoftware/tk-core.svg?branch=master)](http://travis-ci.org/shotgunsoftware/tk-core) [![Build status Windows](https://ci.appveyor.com/api/projects/status/wa0nkvpawf6020fi/branch/master?svg=true)](https://ci.appveyor.com/project/jfboismenu/tk-core/branch/master)
+[![VFX Platform](https://img.shields.io/badge/vfxplatform-2018-yellow.svg)](http://www.vfxplatform.com/)
```

Comment:
![screen shot 2017-09-28 at 11 18 52](https://user-images.githubusercontent.com/8126447/30974912-dfc4760a-a43e-11e7-86a1-3f74cd381c87.png)

Reply:
Clicking on the vfxplatform badge takes you to vfxplatform.com. Clicking on the doc/reference badge takes you to https://developer.shotgunsoftware.com/tk-core

Reply:
Soooooooooo awesome!!!!! ❤️ 💯

Reply:
love the docs link too - super useful! Maybe we could add that to the fws and API too? such a quick and helpful tweak.